Solutions/SAP BTP/Analytic Rules/BTP - Malware detected in BAS dev space.yaml

```diff
@@ -0,0 +1,40 @@
+id: 31997e9a-7447-47f3-8208-4f5d7efe497c
+kind: Scheduled
+name: BTP - Malware detected in BAS dev space
+description: Identifies instances of malware detected using SAP internal malware agent
+  within Business Application Studio dev spaces.
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: SAPBTPAuditLog
+    dataTypes:
+      - SAPBTPAuditLog_CL
+queryFrequency: 15m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - ResourceDevelopment
+  - Execution
+  - Persistence
+relevantTechniques:
+  - T1584
+  - T1072
+  - T0873
+query: |
+  SAPBTPAuditLog_CL
+  | where Message.user =~ "malware_agent"
+  | extend MessageData = parse_json(tostring(Message.data))
+  | extend ClusterID = MessageData.clusterID, WorkspaceID = MessageData.wsID, Message = MessageData.message
+  | project UpdatedOn, ClusterID, WorkspaceID, Message, Tenant, SpaceId, Category, CloudApp = "SAP BTP"
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+entityMappings:
+  - entityType: CloudApplication
+    fieldMappings:
+      - identifier: Name
+        columnName: CloudApp
```

Comment:
Can we also map Malware details here?

Reply:
Unfortunately not, SAP does not provide this level of detail.